public_html/wp-content/mu-plugins/blocks/source/_z-index.scss

```
@@ -0,0 +1,8 @@
+.wordcamp__edit-placeholder .components-placeholder__fieldset {
```

```suggestion
/*
 * When adding elements here, make sure you leave a comment fully explaining which other element they're conflicting with.
 */

.wordcamp__edit-placeholder .components-placeholder__fieldset {
```

`SHOULD` - It'd be good to add a note to the top of this file, similar to the one in the G file that you linked to, asking that any additions here be explained with comments, since comments are as normal in CSS as they should be.

---

👍

---

Done in https://github.com/WordPress/wordcamp.org/pull/123/commits/b03b01398481daf879cd95966a202885ac6b14a7